src/Traits/AvailableActionTrait.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Opstalent\CrudBundle\Traits;
+
+/**
+ * Trait AvailableActionTrait
+ * @author Szymon Kunowski <szymon.kunowski@gmail.com>
+ * @package Opstalent\CrudBundle
+ */
+trait AvailableActionTrait
+{
+    /**
+     * @var array
+     */
+    protected $availableActions = [
+        'addable',
+        'getable',
+        'listable',
+        'editable'
+    ];
+
+    /**
+     * @param array $availableActions
+     * @return $this
+     */
+    protected function setAvailableActions(array $availableActions)
```

Comment:
The method very similar to setActions in ActionTrait but different implementation - is good reason for that?

Reply:
yes (talked personally)